openpype/tools/ayon_loader/ui/products_delegates.py

```diff
@@ -189,3 +195,53 @@ def initStyleOption(self, option, index):
         value = index.data(PRODUCT_IN_SCENE_ROLE)
         color = self._colors.get(value, self._default_color)
         option.palette.setBrush(QtGui.QPalette.Text, color)
+
+
+class AvailabilityDelegate(QtWidgets.QStyledItemDelegate):
+    """
+        Prints icons and downloaded representation ration for both sides.
+    """
+
+    def __init__(self, parent=None):
+        super(AvailabilityDelegate, self).__init__(parent)
+
+    def paint(self, painter, option, index):
+        super(AvailabilityDelegate, self).paint(painter, option, index)
+        option = QtWidgets.QStyleOptionViewItem(option)
+        option.showDecorationSelected = True
+
+        active_icon = index.data(ACTIVE_SITE_ICON_ROLE)
+        remote_icon = index.data(REMOTE_SITE_ICON_ROLE)
+
+        availability_active = f"{index.data(SYNC_ACTIVE_SITE_AVAILABILITY)}/{index.data(REPRESENTATIONS_COUNT_ROLE)}"  # noqa
+        availability_remote = f"{index.data(SYNC_REMOTE_SITE_AVAILABILITY)}/{index.data(REPRESENTATIONS_COUNT_ROLE)}"  # noqa
+
+        if availability_active is None or availability_remote is None:  # group lines
+            return
+
+        idx = 0
+        height = width = 24
+        for value, icon_def in [(availability_active, active_icon),
+                                (availability_remote, remote_icon)]:
+            icon = QtGui.QIcon(icon_def["path"])
```

Comment:
Add import
```
from openpype.tools.ayon_utils.widgets import get_qt_icon
```
and use
```suggestion
            icon = get_qt_icon(icon_def)
```